src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -544,6 +546,15 @@ export class HaConfigDevicePage extends LitElement {
 
                 </div>
           </div>
+          ${
+            deviceAlerts.length
+              ? html`
+                  <div class="alerts fullwidth" slot="alerts">
```

Comment:
There is no `alerts` slot is there?
```suggestion
                  <div class="fullwidth">
```